rpa-bot/1.1/modules/ROOT/pages/configuration.adoc

```diff
@@ -1,8 +1,8 @@
 = Configuring RPA Bot
 
-Use the RPA Bot Configurator application to change the default settings of a bot. You can find the application in the start menu under *MuleSoft* > *RPA Bot Configurator*.
+Use the RPA Bot Configurator application to change the default settings of a bot. To find the application navigate to *MuleSoft* > *RPA Bot Configurator*.
 
-The RPA Bot Configuration tool has five different configuration panels:
+The RPA Bot Configuration tool has five configuration panels:
```

Comment:
You might want to avoid a number there in case more panels show up in the future.